docs/organizations/managing-security-and-risk.md

```diff
@@ -104,6 +104,37 @@ The following table defines item priorities and days to fix the underlying Codac
 | Medium                  | Medium        | 90            |
 | Minor                   | Low           | 120           |
 
+## Access management<!-- Rename this. We're not really only managing access, are we? -->
+
+Organization owners can assign the Security Manager role to organization members, as well as revoke it. Security managers share some permissions with organization owners:
+
+|Permission|Organization Owner|Security Manager|
+|---|---|---|
+|Access Security and Risk Management|Yes|Yes|
+|Access originating items (Codacy repositories, Jira issues)|Yes<!-- TODO admin may nothave access here? Confirm -->|Inherits original permissions|
+|Manage integrations|Yes|<!-- TODO confirm if security managers cans manage integrations -->|
+|Assign and revoke the Security Manager role|Yes|No|
+
+To assign the Security Manager role to a user:
+
+1.  Access the **Security and Risk Management configuration** page.
```

Comment:
The page actually uses sentence case for the title (which is a good thing! :star:). :nerd:


```suggestion
1.  Access the **Security and risk management configuration** page.
```